src/EventStore.Core/Services/Transport/Grpc/Uuid.cs

```diff
@@ -17,11 +12,11 @@ struct Uuid : IEquatable<Uuid> {
 		public static Uuid Parse(string value) => new Uuid(value);
 		public static Uuid FromInt64(long msb, long lsb) => new Uuid(msb, lsb);
 
-		internal static Uuid FromDto(Shared.UUID dto) {
+		internal static Uuid FromDto(Client.Shared.UUID dto) {
```

Comment:
Is this namespace right?

Reply:
Doesn't matter, yet. They are not set over the wire